test/unit-complete/main.tf

```diff
@@ -18,7 +57,48 @@ module "test" {
     "projectOwner:${local.project_id}",
     "projectEditor:${local.project_id}",
     "projectViewer:${local.project_id}",
+    "computed:service-account-name@${local.project_id}.iam.gserviceaccount.com"
   ]
 
+  computed_members_map = {
+    myserviceaccount = "serviceAccount:${module.test-sa.service_account.email}"
+  }
+
+  role = "roles/run.admin"
+
   authoritative = false
+
+  policy_bindings = null # default
+
+  # add most/all other optional arguments
+  module_enabled = true # default
+}
+
+module "test3" {
+  source = "../.."
+
+  # add all required arguments
+  service  = "my-cloud-run-service"
+  location = var.gcp_region
+
+  # add all optional arguments that create additional/extended resources
+  computed_members_map = {
+    myserviceaccount = "serviceAccount:${module.test-sa.service_account.email}"
+  }
+
+  role = null # default
+
+  authoritative = true # default
```

Comment:
```suggestion
```

Reply:
I am curious why we do not want the default values stated explicitly in the `unit-complete`. I think it is good to show all values that are being tested in `unit-complete`. For other tests, I don't think it is relevant.

Reply:
Thanks for the explanation. The idea is by not stating the defaults in the tests is that you can just refer to the documentation. It is not necessary to be able to read the test without referring to the documentation.